meson.build

```diff
@@ -30,6 +30,8 @@ src = [	'src/DataProtocol.cpp',
 	'src/Regulator.cpp',
 	'src/Settings.cpp',
 	'src/UdpDataProtocol.cpp',
+	'src/vsPinger.cpp',
```

Comment:
These includes should only be for !novs builds.

Look around line 101-105. Somewhere in there. Do not include them at this level.